backstory/src/main/webapp/perspective-demo.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Perspective</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato" />
+    <script src="perspective.js"></script>
+  </head>
+
+  <body class="vertical-flex-container" id="perspective-page">
+    <div class="content vertical-flex-container" id="perspective-content">
+      <h1 class="header blue-header">Check Toxicity</h1>
+
+      <form class="vertical-flex-container" id="perspective-form" onsubmit="printToxicity()">
+        <textarea id="text-for-analysis" name="text"></textarea>
+        <button type="submit">Submit</button>
+      </form>
+
+      <div id="toxicity-percentage"><p>Percentage goes here</p></div>
```

Comment:
We talked about this offline... should the Perspective servlet just return the text, if its toxicity is below some threshold?

If not, what object will be responsible for deciding whether the gpt2 text is okay to show?

Reply:
At this point for the demo (and since we'll be using this page in future as you discussed to check Perspective is working and the attributes of certain text), I thought it was best to just have it return all of the attribute scores. I figured in the flow of the work, a different servlet file would decide (possibly the filtering servlet file) because we also later wanted to add other filters than Perspective. As this page is intended to let us test what scores Perspective produces for different text, I think it would reduce its use if it only showed scores under a certain percentage. I'm definitely open to changing this though.

Reply:
We discussed this more offline. This servlet will probably be kept in the demo stage, and since we've redesigned the way the projects work (not as many servlets involved anymore), this works for the demo, but this servlet will not be used in the MVP.

Reply:
Done